engine/src/main/java/com/ibm/engine/utils/DetectionStoreLogger.java

```diff
@@ -162,7 +162,7 @@ private String getDetectionValueContextMessage(
 
     @Nonnull
     String getFormattedNumericString(
-            @Nonnull int hashInt, @Nonnull boolean canBeNegative, @Nullable Integer maxCharacters) {
+            int hashInt, boolean canBeNegative, @Nullable Integer maxCharacters) {
```

Comment:
@hugoqnc is there a reason, why this annotation should be removed?

Reply:
Primitive data types cannot hold null values anyways, and on VScode with "null analysis" enabled, it is an error (not a warning)!